src/containers/Navbar/Navbar.jsx

```diff
@@ -290,8 +291,6 @@ class MenuExampleTabularOnTop extends Component {
     if (result) {
       this.changeValidation('city_zip_state', true);
       await this.setState({ usernameSignIn: this.state.username, passwordSignIn: this.state.password });
```

Comment:
Why use `await` for `this.setState({...})`. This seems to be not a function call that returns a promise.